pgmpy/estimators/BayesianEstimator.py

```diff
@@ -150,13 +153,16 @@ def estimate_cpd(
             )
             pseudo_counts = np.ones(cpd_shape, dtype=float) * alpha
         elif prior_type == "dirichlet":
-            pseudo_counts = np.array(pseudo_counts)
-            if pseudo_counts.shape != cpd_shape:
-                raise ValueError(
-                    "The shape of pseudo_counts for the node: {node} must be of shape: {shape}".format(
-                        node=node, shape=str(cpd_shape)
+            if isinstance(pseudo_counts, list):
+                pseudo_counts = np.array(pseudo_counts)
+                if pseudo_counts.shape != cpd_shape:
+                    raise ValueError(
+                        "The shape of pseudo_counts for the node: {node} must be of shape: {shape}".format(
+                            node=node, shape=str(cpd_shape)
+                        )
                     )
-                )
+            else:
+                pseudo_counts = np.ones(cpd_shape, dtype=int) * pseudo_counts
```

Comment:
@loudly-soft Sorry for the late reply. This line seems to be the culprit: https://travis-ci.org/github/pgmpy/pgmpy/jobs/685246434#L522. Also, you can run specific tests on your system to debug the problem using: `nosetests pgmpy/tests/test_estimators/test_BayesianEstimators.py:TestBayesianEstimator.test_get_parameters2`